### PR TITLE
Edit pass for exchange online migration article DO NOT MERGE

### PR DESCRIPTION
**Pull request by jdmartinez36**
Edit pass for the Exchange Unified Messaging Online migration support article.

Changed the headings in the "Voicemail migration steps" section. This was constructed as an ordered list, but didn't seem to fit the MS Style Guide's definition of what an ordered list should be. I changed the step format to standard level-3 headings and changed the level-2 heading to "Voicemail migration guidelines." 

I changed some heading levels based on whether the information looked like it was separate from the previous heading (not a subheading), but the author should verify.

The "Known issues" section will need a careful review. This is where I found the most inconsistencies in language. I changed the issues to use standard headings and edited them to describe the issues themselves, rather than the workarounds. This could also help with SEO. If the author did not intend this, I would recommend changing the main H3 to "Workarounds for known issues," and I would revert the headings back to the original.

In "Greeting inconsistencies" of "Known issues", there is what appears to be a Powershell cmdlt. I changed the text format to match a cmdlt in a previous section, but the author will want to verify that's it's not some other kind of code.

All other edits were minor and made for clarity and style.

### CODE REVIEW

**Comment by jdmartinez36** [comment]
@tiburd 
Ready for review.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md [hunk 1]
```diff
@@ -217,44 +219,49 @@ The following is an overview of end-user voicemail migration experience.
 |Access to previous messages | No change<br>Users have access to their previous voicemail messages in all supported endpoints. |
 |Receiving VM in outlook, SFB Apps| No change<br>Users continue to receive their voicemail messages in all supported endpoints. |
 |Transcription | Enhanced<br>CVM transcription has a much higher accuracy rate and supported languages than ExchUMO. |
-|User setting | New experience<br>Users are able to change their preferences from a User Setting Portal (USP). Users can access their USP from a hyperlink in their voicemail email, or the User-Settings button on their SFB client; https://aka.ms/vmsettings.   
+|User setting | New experience<br>Users are able to change their preferences from a User Setting Portal (USP). Users can access their USP from a hyperlink in their voicemail email, or the User-Settings button on their SFB client; https://aka.ms/vmsettings.
  |Features| Please see the feature-set comparison for details. |
 |Outlook rules for VM messages | No change<br>Previously created rules will apply to CVM messages after migration.
  |
```
Comment (JohanFreelancer9) [inline Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md:225]:
```suggestion
```

File: Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md [hunk 2]
```diff
@@ -217,44 +219,49 @@ The following is an overview of end-user voicemail migration experience.
 |Access to previous messages | No change<br>Users have access to their previous voicemail messages in all supported endpoints. |
 |Receiving VM in outlook, SFB Apps| No change<br>Users continue to receive their voicemail messages in all supported endpoints. |
 |Transcription | Enhanced<br>CVM transcription has a much higher accuracy rate and supported languages than ExchUMO. |
-|User setting | New experience<br>Users are able to change their preferences from a User Setting Portal (USP). Users can access their USP from a hyperlink in their voicemail email, or the User-Settings button on their SFB client; https://aka.ms/vmsettings.   
+|User setting | New experience<br>Users are able to change their preferences from a User Setting Portal (USP). Users can access their USP from a hyperlink in their voicemail email, or the User-Settings button on their SFB client; https://aka.ms/vmsettings.
  |Features| Please see the feature-set comparison for details. |
 |Outlook rules for VM messages | No change<br>Previously created rules will apply to CVM messages after migration.
```
Comment (JohanFreelancer9) [inline Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md:224]:
```suggestion
|Outlook rules for VM messages | No change<br>Previously created rules will apply to CVM messages after migration. |
```

File: Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md [hunk 3]
```diff
@@ -19,89 +19,91 @@ description: "Microsoft is retiring the Exchange Unified Messaging Online (ExchU
...
 > [!Note]
-> Customers who are using any of the Exchange Server on-premises with Unified Messaging are not affected. 
+> Customers who are using any of the Exchange Server on-premises with Unified Messaging are not affected.
 
 Learn more about the user and admin experience impact in [User experience impact](#user-experience-impact).
 
 ## Migration plan overview
 
-Microsoft has identified various customer deployments that are consuming features from ExchUMO and will be helping customers migrate based on the following plan. 
+Microsoft has identified various customer deployments that are consuming features from ExchUMO and will be helping customers migrate based on the following plan.
```
Comment (JohanFreelancer9) [inline Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md:43]:
```suggestion
Microsoft has identified various customer deployments that are consuming features from ExchUMO and will be helping customers to migrate, based on the following plan.
```

File: Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md [hunk 4]
```diff
@@ -19,89 +19,91 @@ description: "Microsoft is retiring the Exchange Unified Messaging Online (ExchU
...
-    
-    Admins can start migrating their auto attendants from ExchUMO to the Cloud auto attendant at any time. See [Set up a Cloud auto attendant](https://docs.microsoft.com/microsoftteams/create-a-phone-system-auto-attendant) for more details. Microsoft continues to deliver additional auto attendant capabilities that customers may consider required for their migration, admins should evaluate the feature set and migrate their auto attendant instances accordingly. For feature-list comparison, see the [ExchUMO and Azure cloud-based services feature matrix](#exchumo-and-azure-cloud-based-services-feature-matrix).
+If you do not have a Skype for Business hybrid topology established, you need to do that to enable a smooth migration of your voicemail users. See [Configure Skype for Business hybrid](../../SfbHybrid/hybrid/configure-federation-with-skype-for-business-online.md) for more details.
 
-4. **Plan for your voicemail post-migration validation and testing**
+> [!Note]
+> You do not need to migrate your users online for the voicemail service migration. However, for on-premises users to leverage Phone System voicemail service, a hybrid topology must be established.
```
Comment (JohanFreelancer9) [inline Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md:62]:
```suggestion
> You do not need to migrate your users online for the voicemail service migration. However, for on-premises users to leverage the Phone System voicemail service, a hybrid topology must be established.
```

File: Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md [hunk 5]
```diff
@@ -217,44 +219,49 @@ The following is an overview of end-user voicemail migration experience.
...
 
-**Disable Subscriber Access after migration to avoid greeting inconsistency**
-Subscriber access might continue to work for your tenant until the service is completely retired, even after all of your users has been migrated to Cloud Voicemail. To avoid user confusion and inconsistent experience, please disable subscriber access since greetings changed after migration from there will not take affect. To do that, remove the EXUM contact for each subscriber access line using Get-CsExUmContact | ?{$_.IsSubscriberAccess -eq $true} | Remove-CsExUmContact 
+An issue was identified during quality assurance with the "Transfer out to PSTN number" feature, which will not be fixed in time for customers to start migrating off the Exchange UMO service before its scheduled retirement date of Feb 28th, 2020. As a workaround, administrators can transfer auto attendant callers to an on-premise virtual user with an active Call Forward setting to the desired PSTN phone number or RGS phone number. The expected experience is:
```
Comment (JohanFreelancer9) [inline Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md:245]:
```suggestion
An issue was identified during quality assurance with the "Transfer out to PSTN number" feature, which will not be fixed in time for customers to start migrating off the Exchange UMO service before its scheduled retirement date of Feb 28th, 2020. As a workaround, administrators can transfer auto attendant callers to an on-premises virtual user with an active Call Forward setting to the desired PSTN phone number or RGS phone number. The expected experience is:
```

File: Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md [hunk 6]
```diff
@@ -217,44 +219,49 @@ The following is an overview of end-user voicemail migration experience.
...
- 
-Expected Experience
-- Administrators do not need to license the virtual user, since this is a workaround solution 
-- Administrators can manipulate the caller ID that the PSTN receiver will see by assigning the desired number to the virtual user, or using the SBC digit manipulation capabilities
-- PSTN Callers will not experience any delay during the call transfer, and they will continue to see the caller ID of the auto attendant after the transfer is successful  
+- Administrators will not need to license the virtual user since this is a workaround solution.
+- Administrators can manipulate the caller ID that the PSTN receiver sees by assigning the desired number to the virtual user or using the SBC digit manipulation capabilities.
+- PSTN Callers will not experience any delay during the call transfer and will continue to see the caller ID of the auto attendant after the transfer is successful.
```
Comment (JohanFreelancer9) [inline Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md:249]:
```suggestion
- PSTN callers will not experience any delay during the call transfer and will continue to see the caller ID of the auto attendant after the transfer is successful.
```

File: Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md [hunk 7]
```diff
@@ -217,44 +219,49 @@ The following is an overview of end-user voicemail migration experience.
...
 To learn more about auto attendants, see [Set up a Cloud auto attendant](https://docs.microsoft.com/microsoftteams/create-a-phone-system-auto-attendant).
 
-#### Known issues
+### Known issues
+
+#### Greeting inconsistencies
+
+Subscriber access might continue to work for your tenant until the service is completely retired, even after all your users have been migrated to Cloud Voicemail. To avoid user confusion and an inconsistent experience, disable the subscriber access since the greetings change after migration. To do that, remove the EXUM contact for each subscriber access line using `Get-CsExUmContact | ?{$_.IsSubscriberAccess -eq $true} | Remove-CsExUmContact`.
+
+#### Auto Attendant Call Transfer to PSTN
```
Comment (JohanFreelancer9) [inline Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md:241]:
```suggestion
#### Auto attendant call transfer to PSTN
```

File: Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md [hunk 8]
```diff
@@ -217,44 +219,49 @@ The following is an overview of end-user voicemail migration experience.
...
 |Outlook rules for VM messages | No change<br>Previously created rules will apply to CVM messages after migration.
  |
 
-#### User management and provisioning in CVM
+### User management and provisioning in CVM
 
 New Skype for Business users will be automatically provisioned for Cloud voicemail when created. No additional admin work or license is required to provision new voicemail users. See [Set up Cloud Voicemail](/microsoftteams/set-up-phone-system-voicemail) to learn about policy management for existing and new users.
 
-#### Admin Auto Attendant management experience
+### Admin Auto Attendant management experience
```
Comment (JohanFreelancer9) [inline Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md:231]:
```suggestion
### Admin auto attendant management experience
```

**Comment by tiburd** [comment]
Hi, Heidi/@MicrosoftHeidi,

We’ve completed the edit pass of this article. Please review it and let me know if we’ve changed the technical meaning anywhere. You have five business days for the review (If you need longer, let me know). 

When your review is complete, and you’re satisfied with the changes, our team will do the final merge to confirm there are no remaining PR issues.

The editor noted that he made some changes that might be a little heavy for a light copy edit pass, so please review carefully. He detailed the changes at the top of the PR.

Thanks!

**Comment by SerdarSoysal** [comment]
I reviewed this PR. Approving merge to live. @MicrosoftHeidi FYI.

**Comment by SerdarSoysal** [comment]
@tiburd Please see above

**Pull request merged**